### PR TITLE
HostGetString: Actually fill a string with data

### PR DESCRIPTION
**Pull request by shuffle2**

### CODE REVIEW

File: Source/Core/Core/PowerPC/MMU.cpp [hunk 1]
```diff
@@ -637,6 +637,7 @@ std::string HostGetString(u32 address, size_t size)
 		u8 res = HostRead_U8(address);
 		if (!res)
 			break;
+		s.append(1, (char)res);
```
Comment (Tilka) [inline Source/Core/Core/PowerPC/MMU.cpp]: I'd like to bikeshed this into `s += (char)res;`.

Comment (lioncash) [inline Source/Core/Core/PowerPC/MMU.cpp]: I'd like to bikeshed the bikeshed into
`s += static_cast<char>(res);`

Comment (Tilka) [inline Source/Core/Core/PowerPC/MMU.cpp]: Oh come on XD

**Comment by JosJuice** [comment]
@dolphin-emu-bot rebuild

**Pull request merged**